modules/ROOT/pages/plugins/premium/a11ychecker.adoc

```diff
@@ -28,7 +28,7 @@ tinymce.init({
   toolbar: 'a11ycheck'
 });
 ----
-
+[[accessiblityrules]]
```

Comment:
```suggestion

[[accessiblityrules]]
```

Reply:
The spelling for 'accessibility' needs to be updated on link and anchor
suggest:
[[accessibilityrules]]